gradio/components.py

```diff
@@ -723,6 +729,12 @@ def get_config(self):
             **IOComponent.get_config(self),
         }
 
+    def get_random_value(self):
+        return random.randrange(int(self.minimum), int(self.maximum))
```

Comment:
Is there a reason we are restricting to integers? If the minimum is 0 and the maximum is 1, for example, a float would make more sense.

I think the right thing to do here is to do a `random.choice()` from amongst the possible values of the slider. Something like `value = random.choice(range(self.minimum, self.maximum, self.step))`

Reply:
Yea good point! This is trickier than I thought hehe.

The number of possible values can be huge so I don't think generating all possible values just to pick one is the best approach.

We might be better off generating a random multiple of step from the set of all possible multiples allowed by the min and max.

Something like this:

```python
def get_random_value(self,):
        # Goal is to generate a possible value of the slider
        # without generating all possible values
        # We randomly pick a step among all possible steps and add  to the minimum
        n_steps = int((self.maximum - self.minimum) / self.step)
        step = random.randint(0, n_steps)
        value = self.minimum + step * self.step
        return value
```